apps/developer-portal/src/configs/routes.ts

```diff
@@ -149,6 +149,28 @@ const DASHBOARD_LAYOUT_ROUTES: RouteInterface[] = [
         protected: true,
         showOnSidePanel: true
     },
+    {
+        children: [
+            {
+                component: lazy(() => import("../pages/oidc-scopes/oidc-scopes-edit")),
+                exact: true,
+                icon: "overview",
```

Comment:
`icon` typings have changed, and now it accept a `GenericIconProps` type. These changes will be reflected once the latest master branch is rebased/merged.

```suggestion
                icon: {
                    icon: SidePanelIcons.overview
                },
```